examples/global_variables_test.py

```diff
@@ -0,0 +1,37 @@
+"""Test file for global variables checker
+"""
+
+
+def method():
```

Comment:
Use the term "function", not "method". Methods are functions which are defined in a class, bound to an object.